src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1511,7 +1511,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         BuildInParallel="$(BuildInParallel)"
         Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration); %(_MSBuildProjectReferenceExistent.SetPlatform); ReferringTargetFramework=$(TargetFrameworkMoniker)"
         ContinueOnError="!$(BuildingProject)"
-        RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)">
+        RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove);TargetFramework">
```

Comment:
Also, wanted to get in the change that matches what got the SDK tests happy again and not code more stuff blind. I'd like to start adding tests and would be more comfortable making further changes if the tests were in.